.github/workflows/edge.yml

```diff
@@ -153,92 +203,25 @@ jobs:
           name: 'trivy-results-${{ matrix.image }}.sarif'
           path: 'trivy-results-${{ matrix.image }}.sarif'
         if: always()
-    strategy:
-      matrix:
-        image: [debian, alpine, opentracing, ubi]
 
   smoke-tests:
     name: Smoke Tests
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-20.04
     needs: [binary, unit-tests]
-    if:
-      github.event.pull_request.head.repo.full_name == 'nginxinc/kubernetes-ingress' ||
-      github.event_name == 'push'
+    if: github.event_name == 'push' || github.event_name == 'pull_request'
     strategy:
       matrix:
         include:
-          - os: ubuntu-20.04
-            image: debian
-            tag: ${{ github.sha }}
-            marker: 'ingresses'
-            type: oss
-            ic-type: nginx-ingress
-          - os: ubuntu-20.04
-            image: alpine
-            tag: ${{ github.sha }}
-            marker: 'vsr'
-            type: oss
-            ic-type: nginx-ingress
-          - os: ubuntu-20.04
-            image: opentracing
-            tag: ${{ github.sha }}
-            marker: 'vs'
-            type: oss
-            ic-type: nginx-ingress
-          - os: ubuntu-20.04
-            image: ubi
-            tag: ${{ github.sha }}
-            marker: 'ts'
-            type: oss
-            ic-type: nginx-ingress
-          - os: ubuntu-20.04
-            image: debian
-            tag: ${{ github.sha }}
-            marker: 'policies'
-            type: oss
-            ic-type: nginx-ingress
-          - os: ubuntu-20.04
-            image: ubi-plus
-            tag: ${{ github.sha }}
+          - image: debian
             marker: 'ingresses'
-            type: plus
-            ic-type: nginx-plus-ingress
-          - os: ubuntu-20.04
-            image: debian-plus
-            tag: ${{ github.sha }}
+          - image: alpine
```

Comment:
Should we run all tests on `debian` only? because of `runs-on: ubuntu-20.04`?

Reply:
`runs-on: ubuntu-20.04` is the just the runner OS - we always used ubuntu-20.04 runner for all of our jobs. The image is which IC image version we are using in the KinD cluster.